apps/pipeline/test/integration/pipeline/writer/table_writer_test.exs

```diff
@@ -142,67 +157,79 @@ defmodule Pipeline.Writer.TableWriterTest do
+        }
+      }
 
       assert :ok = TableWriter.write([datum], table: dataset.technical.systemName, schema: schema)
 
       eventually(fn ->
+        query = "select * from foo__baz"
+
         result =
-          "select * from foo__baz"
-          |> Prestige.execute()
-          |> Prestige.prefetch()
+          session
+          |> Prestige.execute!(query)
+          |> Prestige.Result.as_maps()
 
         assert result == [expected]
       end)
     end
   end
 
   describe "compact/1" do
-    test "compacts a table without changing data" do
+    test "compacts a table without changing data", %{session: session} do
       sub = [%{name: "three", type: "boolean"}]
-      schema = [%{name: "one", type: "list", itemType: "integer"}, %{name: "two", type: "map", subSchema: sub}]
+      schema = [%{name: "one", type: "list", itemType: "decimal"}, %{name: "two", type: "map", subSchema: sub}]
       dataset = TDG.create_dataset(%{technical: %{schema: schema, systemName: "a__b"}})
 
       TableWriter.init(table: dataset.technical.systemName, schema: schema)
 
-      Enum.each(1..10, fn n ->
+      Enum.each(1..15, fn n ->
         payload = %{"one" => [n], "two" => %{"three" => false}}
         datum = TDG.create_data(%{dataset_id: dataset.id, payload: payload})
         TableWriter.write([datum], table: dataset.technical.systemName, schema: schema)
       end)
 
       eventually(fn ->
-        assert [[10]] =
-                 "select count(1) from #{dataset.technical.systemName}"
-                 |> Prestige.execute()
-                 |> Prestige.prefetch()
+        query = "select count(1) from #{dataset.technical.systemName}"
+
+        result =
+          session
+          |> Prestige.query!(query)
+
+        assert result.rows == [[15]]
       end)
 
-      assert :ok = TableWriter.compact(table: dataset.technical.systemName)
+      assert :ok == TableWriter.compact(table: dataset.technical.systemName)
 
       eventually(fn ->
-        assert [[10]] =
-                 "select count(1) from #{dataset.technical.systemName}"
-                 |> Prestige.execute()
-                 |> Prestige.prefetch()
+        query = "select count(1) from #{dataset.technical.systemName}"
+
+        result =
+          session
+          |> Prestige.query!(query)
+
+        assert result.rows == [[15]]
       end)
     end
 
-    test "fails without altering state if it was going to change data" do
+    test "fails without altering state if it was going to change data", %{session: session} do
       allow Compaction.measure(any(), any()), return: {6, 10}, meck_options: [:passthrough]
 
       schema = [%{name: "abc", type: "string"}]
       dataset = TDG.create_dataset(%{technical: %{schema: schema, systemName: "xyz"}})
 
       TableWriter.init(table: dataset.technical.systemName, schema: schema)
 
-      Enum.each(1..10, fn n ->
+      Enum.each(1..15, fn n ->
```

Comment:
any particular reason we increased the number of rows here and above? ~Jeff

Reply:
It seems to be an issue with prestige, opened an [issue](https://github.com/smartcitiesdata/prestige/issues/19) for it.